### PR TITLE
Add private-can-be-overridden bug to buglist

### PR DESCRIPTION
**Pull request by Marenz**
refs #8417

### CODE REVIEW

**Comment by leonardoalt** [comment]
Please rebase, should fix the travis problem

File: docs/bugs.json [hunk 1]
```diff
@@ -10,6 +10,14 @@
             "yulOptimizer": true
         }
     },
+    {
+        "name": "privateCanBeOverridden0.5",
+        "summary": "Private methods can be overridden by inheriting classes.",
+        "description": "Private methods can be overridden by inheriting classes.",
```
Comment (chriseth) [inline docs/bugs.json]:
```suggestion
        "description": "While private methods of base contracts are not visible and cannot be called directly from the derived contract, it is still possible to declare a function of the same name and type and thus change the behaviour of the base contract's function.",
```

File: docs/bugs.json [hunk 2]
```diff
@@ -10,6 +10,14 @@
             "yulOptimizer": true
         }
     },
+    {
+        "name": "privateCanBeOverridden0.5",
+        "summary": "Private methods can be overridden by inheriting classes.",
```
Comment (chriseth) [inline docs/bugs.json]:
```suggestion
        "summary": "Private methods can be overridden by inheriting contracts.",
```

File: docs/bugs.json [hunk 3]
```diff
@@ -10,6 +10,14 @@
             "yulOptimizer": true
         }
     },
+    {
+        "name": "privateCanBeOverridden0.5",
```
Comment (chriseth) [inline docs/bugs.json]:
```suggestion
        "name": "privateCanBeOverridden",
```

**Comment by Marenz** [comment]
Rebased and applied suggestions

**Review by chriseth — APPROVED** [review]

**Pull request merged**